lib/exandra.ex

```diff
@@ -218,6 +218,34 @@ defmodule Exandra do
                         Xandra.Cluster
                       )
 
+  @xandra_prepare_opts [
```

Comment:
Instead of duplicating these lists and the logic here, can we create a `Xandra.Connection.split_prepare_and_execute_options/1` and use that here?

Reply:
Done

Also fixed :telemetry_metadata and tests failing due to duplicate keys